source/object-model/classes-and-instances.md

```diff
@@ -17,8 +21,8 @@ You can also create a _subclass_ from any existing class by calling
 its `extend()` method. For example, you might want to create a subclass
 of Ember's built-in `Ember.Component` class:
 
-```app/views/person.js
-PersonView = Ember.Component.extend({
+```app/components/person.js
```

Comment:
note to myself for revising:

```
```app/components/todo-item.js
export default Ember.Component.extend({
  classNameBindings: ['isUrgent'],
  isUrgent: true
});
```